sources/packages/web/package-lock.json

```diff
@@ -641,15 +643,15 @@
       }
     },
     "node_modules/@formio/js": {
-      "version": "5.0.1",
-      "resolved": "https://registry.npmjs.org/@formio/js/-/js-5.0.1.tgz",
-      "integrity": "sha512-MVAT9lseEYmdiivVuQRvWxwXpp402YYpStrTkMqi8nvCA/AmFb9Ld5nsFUqQsfzf4jzY5NgefFmoFlSMfTk0Kg==",
+      "version": "5.0.0",
+      "resolved": "https://registry.npmjs.org/@formio/js/-/js-5.0.0.tgz",
+      "integrity": "sha512-Ly3f7q2wo5oFjlECKFtkIMy+Ah9HyAkf6ob70OrF/6MooSPF3hEu9euXouEWOoNdI+Tokr690/GdM2sDZtYzww==",
```

Comment:
Did it downgrade the SDK?

Reply:
Not intentionally. The main package was never upgraded to `5.0.1`. The lock was updated 2 weeks ago when we did the nestjs upgrade (potentially not intentional).
I will revert to keep the changes to minimal.